Sources/Swiftagram/Extensions/RequestExtensions.swift

```diff
@@ -19,7 +19,10 @@ public extension Requestable where Self: QueryComposable & QueryParsable {
         return self.appending(query: key, with: value)
             .prepare { request, response in
                 guard let response = response else { return request }
-                return try? response.get()[keyPath: keyPath].string().flatMap { request.appending(query: key, with: $0) }
+guard let response = try? response?.get() else { return request }
+ return (response[keyPath: keyPath].string() 
+    ?? response[keyPath: keyPath].int().flatMap(String.init))
+        .flatMap { request.appending(query: key, with: $0) }
```

Comment:
I fucked up the indentation.

```suggestion
                guard let response = try? response?.get() else { return request }
                return (response[keyPath: keyPath].string() ?? response[keyPath: keyPath].int().flatMap(String.init))
                    .flatMap { request.appending(query: key, with: $0) }
```